Packs/Microsoft365Defender/Playbooks/playbook-Microsoft_365_Defender_-_Emails_Indicators_Hunt.yml

```diff
@@ -535,16 +533,14 @@ tasks:
     isautoswitchedtoquietmode: false
   "15":
     id: "15"
-    taskid: 6323c0a2-b979-4320-85be-3193ad8ee226
+    taskid: a9b33515-9725-4e37-8188-2120f8fb69e3
     type: regular
     task:
-      id: 6323c0a2-b979-4320-85be-3193ad8ee226
+      id: a9b33515-9725-4e37-8188-2120f8fb69e3
       version: -1
-      name: Retrieve emails data (IPAddress)
-      description: 'Advanced hunting is a threat-hunting tool that uses specially
-        constructed queries to examine the past 30 days of event data in Microsoft
-        365 Defender. Details on how to write queries: https://docs.microsoft.com/en-us/microsoft-365/security/defender/advanced-hunting-query-language?view=o365-worldwide.'
+      name: Retrieve emails data (IPAddress) - EmailEvents table
       script: Microsoft 365 Defender|||microsoft-365-defender-advanced-hunting
+      description: 'This task will retrieve emails data based on the EmailEvents table. It will look for the given IP in the "SenderIPv4" and in the "SenderIPv46" properties.'
```

Comment:
Should this be:
...the given IP in the "SenderIPv4" and in the "**SenderIPv6**" properties.'